src/store/searchResult/index.tsx

```diff
@@ -0,0 +1,49 @@
+import {
+  createSlice,
+  PayloadAction
+} from '@reduxjs/toolkit';
+
+import { Feature } from 'geojson';
+
+export interface SearchResultState {
+  geoJSONFeature: Feature | null;
+  drawerVisibility: boolean;
+}
+
+const initialState: SearchResultState = {
+  drawerVisibility: false,
+  geoJSONFeature: null
+};
+
+export const slice = createSlice({
+  name: 'stylingDrawerVisibility',
```

Comment:
```suggestion
  name: 'searchResultDrawer',
```

Not sure if this change requires other changes as well…